app/views/content/explore-teaching-advisers.md

```diff
@@ -3,8 +3,8 @@ title: Explore teaching advisers
 description: |-
     Find out if teaching is for you with an explore teaching adviser who can help you discover what teaching is really like.
 layout: "layouts/minimal"
-colour: "yellow"
-image: "static/images/content/ttalianne.jpg"
```

Comment:
We should delete this image altogether - it is not used elsewhere (can be done on a separate PR if needed)

@jenhadfield-dfe

Reply:
done - have removed the image